app/views/alternatives/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= render partial: "collectivities/show_banner", locals: { duration: 2 } %>
+
+<div class="fr-col-md-8 fr-my-6w">
+  <%= simple_format t('.caf.title'), { }, wrapper_tag: 'h2' %>
+
+  <%= simple_format t('.caf.description'), { }, wrapper_tag: 'p' %>
+
+  <%= form_with url: collectivity_alternatives_path(@collectivity.siret), scope: :alternative, class: "fr-grid-row fr-grid-row--gutters" do |f| %>
+    <div class="fr-col-12 fr-col-md-8">
+      <%= f.label :allocataire_number, t('.caf.allocataire_number') %>
+      <%= f.text_field :allocataire_number, class: "fr-input fr-mt-1w", required: true %>
+    </div>
+    
+    <div class="fr-col-12 fr-col-md-4">
+      <%= f.label :postal_code, t('.caf.postal_code') %>
+      <%= f.text_field :postal_code, class: "fr-input fr-mt-1w", required: true %>
+    </div>
+
+    <div class="fr-col-12">
+      <%= button_tag type: "submit", class: "fr-btn fr-btn--lg fr-btn--icon-right fr-fi-arrow-right-line" do %>
+        <%= t('.caf.submit') %>
+      <% end %>
+    </div>
+  <% end %>
+</div>
+
+<div class="fr-col-md-8">
+  <%= simple_format t('.msa.title'), { }, wrapper_tag: 'h2' %>
+
+  <%= render partial: "collectivities/alternatives", locals: { only_key: :msa } %>
```

Comment:
ça me paraitrait plus logique de fournir au partial ce sur quoi on veut itérer, ce qui rendrait le partial _alternatives plus lisible 🤔 
```suggestion
  <%= render partial: "collectivities/alternatives", locals: { alternatives: t('collectivities.no_france_connect.alternative_options').slice(:msa) } %>
```

Reply:
pas faux :thu